src/redux/reducers/usersListReducer.js

```diff
@@ -0,0 +1,44 @@
+import {
+  GET_USERS_SUCCESS,
+  GET_USERS_FAIL,
+  START_GETTING_USERS,
+} from '../actions/actionTypes';
+
+const initialState = {
+  users: [],
+  isFetched: false,
+  isFetching: false,
```

Comment:
Actually You are using is Fetching here.

Reply:
@Mubangizi this too has been fixed